picker.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Creates a custom menu in Google Sheets when the spreadsheet opens.
+ */
+function onOpen() {
```

Comment:
we should have a single onOpen function per sheet, so may be merge this with the other one in another file?

Reply:
Yeah, so right now i have made these file picker and local file uploader separately in separate sheets that's why they have separate on-open functions. These on-open functions are mainly for testing because once we integrate these with the project, these won't be executed as add-on menus, instead i'll be calling the function which on-open calls by myself. So, there won't be any on-open then.